packages/core/src/navigation-item/NavigationItem.tsx

```diff
@@ -75,34 +128,69 @@ export const NavigationItem = forwardRef<HTMLDivElement, NavigationItemProps>(
       "--saltNavigationItem-level": `${level}`,
     };
 
+    const isParent = parent || href === undefined; // for backwards compatiblity with original
```

Comment:
This comment is written like a comment on the PR and will lose context once it's merged. It's probably not needed.